rosidl_typesupport_c/resource/action__type_support.cpp.em

```diff
@@ -0,0 +1,124 @@
+// generated from rosidl_typesupport_c/resource/action__type_support.cpp.em
```

Comment:
Same as above comment I don't think this file is needed because no dispatch is needed for actions.

Reply:
True.